internal/core/services/identity.go

```diff
@@ -63,10 +64,11 @@ type identity struct {
 
 	ignoreRHSErrors bool
 	rhsPublisher    reverse_hash.RhsPublisher
+	pubsub          pubsub.Client
```

Comment:
```suggestion
	pubsub          pubsub.Publisher
```
Just a suggestion. Using pubsub.Client is perfectly acceptable. But injecting pubsub.Publisher tells the reader that this service is publishing messages but not listening to them.